Content.Shared/Inventory/InventorySystem.Equip.cs

```diff
@@ -340,6 +359,24 @@ public bool CanEquip(EntityUid actor, EntityUid target, EntityUid itemUid, strin
 
         Transform(removedItem.Value).Coordinates = Transform(target).Coordinates;
 
+        if (!silent && Resolve(removedItem.Value, ref item) && item.UnequipSound != null && _gameTiming.IsFirstTimePredicted)
+        {
+            Filter filter;
+
+            if (_netMan.IsClient)
+                filter = Filter.Local();
+            else
+            {
+                filter = Filter.Pvs(target);
+
+                // don't play double audio for predicted interactions
+                if (predicted)
+                    filter.RemoveWhereAttachedEntity(entity => entity == actor);
+            }
+
+            SoundSystem.Play(filter, item.UnequipSound.GetSound(), target, AudioParams.Default.WithVolume(-2f));
+        }
```

Comment:
wow this is horrific boilerplate, not going to force you to change that here though

Reply:
yeah it is. there is currently no nice way to handle audio prediction. Usually there are separate client/server systems but here I just combined them with `_netMan.IsClient` which also makes it all uglier.